src/client/tablet_client.h

```diff
@@ -298,7 +296,7 @@ class TabletClient : public Client {
 
  private:
     ::openmldb::RpcClient<::openmldb::api::TabletServer_Stub> client_;
-    std::vector<uint64_t> percentile_;
+    std::vector<uint64_t>;
```

Comment:
wrong statement

Reply:
so should I delete line 301 only? @aceforeverd

Reply:
delete this line